requirements.txt

```diff
@@ -5,4 +5,5 @@
 
 # NOTE - for the time being - keep the same packages and versions in the `package_installer_dialog.py`
 onnxruntime-gpu==1.12.1
-opencv-python-headless==4.6.0.66  # opencv-python if you would like to display images while debugging
+opencv-python-headless==4.6.0.66; python_version > '3.9'  # opencv-python if you would like to display images while debugging
```

Comment:
The new version of OpenCV is a little troublemaker for python 3.8.

Reply:
Please note the note above - we need to update packages in `package_installer_dialog.py` too